### DISCUSSION TITLE
Windows Deployment

### DISCUSSION THREAD

**Discussion by chrpai** [dotnet/maui#11716] [Q&A]
https://learn.microsoft.com/en-us/dotnet/maui/windows/deployment/overview?view=net-maui-7.0 says

".NET MAUI currently only allows publishing an MSIX package. You can't yet publish a Windows executable file for distribution."

Whats the future hold for this?  Timeline?    I'm interested in looking at MAUI to replatform my Winforms, WPF and Xamarin applications but not if I can't build a traditional Windows application.

**Comment by xtuzy** [comment]
please search it in issues before ask question, you can find https://github.com/dotnet/maui/issues/11254
